my-app/src/main/java/com/JavaWebServer/app/RESTMethods/Post.java

```diff
@@ -0,0 +1,17 @@
+package com.JavaWebServer.app;
+
+public class Post implements RestMethod {
+  private final String METHODKEY = "POST";
+  private final String ACCEPTED ="HTTP/1.1 200 ok";
+
+  public Post() { }
+
+  public String handleRequest(Request request) {
+    if ((request.getMethod()).equals(METHODKEY)) {
```

Comment:
This doesn't really seem like the proper place to be handling "unhandled" methods. The class is called Post, so you should really only be using this class if you already know your method is POST.